database-commons/src/main/java/io/cdap/plugin/db/batch/sink/AbstractDBSink.java

```diff
@@ -258,15 +262,16 @@ private void validateSchema(Class<? extends Driver> jdbcDriverClass, String tabl
       throw Throwables.propagate(e);
     }
 
-    try (Connection connection = DriverManager.getConnection(connectionString, dbSinkConfig.getConnectionArguments());
-         ResultSet tables = connection.getMetaData().getTables(null, null, tableName, null)) {
-
-      Preconditions.checkArgument(tables.next(),
-                                  "Table %s does not exist. " +
-                                    "Please check that the 'tableName' property " +
-                                    "has been set correctly, and that the connection string %s " +
-                                    "points to a valid database.",
-                                  tableName, connectionString);
+    try (Connection connection = DriverManager.getConnection(connectionString, dbSinkConfig.getConnectionArguments())) {
+      executeInitQueries(connection, dbSinkConfig.getInitQueriesString());
+      try (ResultSet tables = connection.getMetaData().getTables(null, null, tableName, null)) {
+        Preconditions.checkArgument(tables.next(),
```

Comment:
validate methods should throw InvalidStageException or InvalidConfigPropertyException